src/Kafka/CreateKafkaInstance/Stories/storiesHelpers.tsx

```diff
@@ -53,7 +53,13 @@ const AZURE: CloudProviderInfo = {
   ],
 };
 
-export const PROVIDERS: CloudProviderInfo[] = [AWS, AZURE];
+const GCP: CloudProviderInfo = {
+  id: "gcp",
+  displayName: "Google cloud platform",
+  regions: [],
```

Comment:
Could you please add some sample region? So to make the initial example work when selecting GCP.